### DISCUSSION TITLE
Text over images.

### DISCUSSION THREAD

**Discussion by MarcelineTheVampireQueen** [sumatrapdfreader/sumatrapdf#3309] [General]
Just downloaded sumatraPDF and I'm unsure If I'm suppose to be posting this here or not, but my text has been appearing over images in books. The issue is in 2 of the books I have, but I don't know if it's because of the website I got them from or not. Anyone know how to fix this?
![Text issue](https://user-images.githubusercontent.com/126537577/221742361-bd99d489-99c6-4b59-9331-3c4f6ebce566.png)
![text issue 2](https://user-images.githubusercontent.com/126537577/221742371-d66bf1b0-c0bb-4dd2-b29a-96c430d1cb98.png)
![text issue 3](https://user-images.githubusercontent.com/126537577/221742377-3aeb15a4-beaa-4215-8d99-d256cc8a6a7f.png)

**Comment by GitHubRulesOK** [comment]
Hmmmm looks as if its possibly e-book /epub ?? 
may be similar to another issue but developer would need a copy to verify if its same as open MuPDF issue
you can mail him a copy direct email address kkowalczyk@gmail.com if its not to be made public here.
